src/components/datePicker/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { forwardRef } from 'react';
+import { Field, Input, InputProps, useSlotRecipe } from '@chakra-ui/react';
+
+export interface DatePickerProps
+  extends Pick<InputProps, 'onFocus' | 'onBlur' | 'onChange'> {
+  size?: 'md' | 'lg';
+  min?: Date;
+  value?: string;
+  invalid?: boolean;
+}
+
+export const DatePicker = forwardRef<HTMLInputElement, DatePickerProps>(
+  ({ min, ...props }, ref) => {
+    const recipe = useSlotRecipe({ key: 'input' });
+    const [recipeProps] = recipe.splitVariantProps(props);
+    const styles = recipe(recipeProps);
+    return (
+      <Field.Root invalid={props.invalid}>
```

Comment:
Why do we need the `Field` wrapper now? The `input` has the `invalid` style so we should be able to leverage it directly?